src/new/connection.js

```diff
@@ -97,6 +98,18 @@ class Connection extends EventEmitter {
     this.websocket.on('is_online', this.onConnectionChange);
     this.websocket.on('wallet', this.handleWalletMessage);
 
+    this.websocket.on('reload_data', () => {
+      this.emit('reload-data');
```

Comment:
Why do we have to propagate this event? What does it mean for the app?

Reply:
This event is not needed. It was useless, then I added the `beforeReload` callback